client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -109,6 +109,8 @@ export const siteSetupFlow: Flow = {
 					// End of woo flow
 					if ( storeType === 'power' ) {
 						dispatch( recordTracksEvent( 'calypso_woocommerce_dashboard_redirect' ) );
+
+						return exitFlow( `https://${ siteSlug }/wp-admin/admin.php?page=wc-admin` );
```

Comment:
In the `/start` framework a [specific selector](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/steps/woocommerce-install/transfer/install-plugins.tsx#L39) is being used to assemble this url. Do we need to bring this over to the data-store?

Reply:
Ah, didn't know about that! Let me look and see if it's something we need in the site data-store.

Reply:
@andres-blanco I've got a new pr (#63305) which adds selectors for retrieving site options. I'll use those selectors to retrieve the admin url and assemble the url in the confirm step.